src/client/components/AdminPanel/AdminListings.js

```diff
@@ -105,89 +122,61 @@ export const AdminListings = ({ mediaType, setError }) => {
         });
       });
   };
-  useEffect(retrieveSubjectAreas, [recentlySearchedTerm]);
-
-  const handleListingsSearch = event => {
-    setRecentlySearchedTerm(searchTerm);
-    setSelectedSubjectArea('');
-    event.preventDefault();
-  };
+  useEffect(retrieveSubjectAreas, [selectedAcademicTerm]);
 
   return (
     <View>
-      <form name="listingsSearch" onSubmit={handleListingsSearch}>
-        <Grid
-          startAt="medium"
-          vAlign="middle"
-          colSpacing="none"
-          rowSpacing="small"
-        >
-          <Grid.Row>
-            <Grid.Col>
-              <div>
-                <Text>
-                  <strong>Term</strong> (leave empty to show content from all
-                  terms)
-                </Text>
-                &nbsp;
-                <TextInput
-                  renderLabel={
-                    <ScreenReaderContent>
-                      Academic term text field
-                    </ScreenReaderContent>
-                  }
-                  display="inline-block"
-                  width="5rem"
-                  onChange={handleTermInput}
-                />
-                &nbsp;
-                <Button color="primary" onClick={handleListingsSearch}>
-                  Go
-                </Button>
-              </div>
-            </Grid.Col>
-          </Grid.Row>
-          <Grid.Row>
-            <Grid.Col>
-              <SimpleSelect
-                renderLabel="Subject Area"
-                value={selectedSubjectArea}
-                onChange={handleSubjectSelect}
-                visibleOptionsCount={5}
-                width="12rem"
-              >
-                <SimpleSelect.Option id="all" value="">
-                  All
+      <Flex margin="none none none">
```

Comment:
Just curious, does margin="none none none" make any difference?

Reply:
Oh yeah, good catch, forgot to remove it when I was tinkering with the margin